src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBuffer.cs

```diff
@@ -625,6 +625,10 @@ internal SqlBinary SqlBinary
             {
                 if (StorageType.SqlBinary == _type)
                 {
+                    if (IsNull)
```

Comment:
We also need to port this to the netfx SqlBuffer.cs file. The fix in main only had one file because the file had been merged by that point.